src/app/judging/JudgingTable.tsx

```
@@ -21,20 +21,22 @@ export default function JudgingTable({
   >;
 }) {
   const [selectedTeam, setSelectedTeamId] = useState("");
+  const [teamsLeft, setTeamsLeft] = useState(0);
 
   const { currentUser } = useUser();
   const { data: roomData, isFetching: roomIsFetching } = useQuery({
     queryKey: ["RoomForJudge", currentUser.JUDGE_roomId],
     queryFn: async () => {
-      if (!currentUser.JUDGE_roomId) throw Error("No room assigned to judge");
+      if (currentUser.JUDGE_roomId === undefined) return null;
```

returning null here is not good, but again we dont want to throw the error. It expects the data being returned to be of type roomData, so like Schema<Room> or whatever, so we can maybe just return {} as Schema<Room>, and then ahve a subsequent check whenever this roomdata is referenced to handle cases where it is empty.